src/handle_action.cpp

```diff
@@ -737,6 +737,11 @@ static void smash()
         if( maybe_corpse.is_corpse() && maybe_corpse.damage() < maybe_corpse.max_damage() &&
             maybe_corpse.can_revive() ) {
             // do activity forever. ACT_PULP stops itself
+            if( maybe_corpse.get_mtype()->bloodType().obj().has_acid ) {
```

Comment:
```suggestion
            if( maybe_corpse.get_mtype()->bloodType()->has_acid ) {
```

Reply:
I'll change this, but know that's the syntax I found in activity_handlers.cpp line 1953.  It may need to be updated.

Reply:
Oh yeah, you'll find all sort of different variants using one format or the other. As far as I know, this is the preferred one. They both do the same thing, so no big deal.